ConcernsCaseWork/ConcernsCaseWork.Tests/Security/RbacManagerTests.cs

```diff
@@ -21,8 +23,9 @@ public async Task WhenGetSystemUsers_Return_Users()
 			// arrange
 			var mockUserRoleCachedService = new Mock<IUserRoleCachedService>();
 			var mockLogger = new Mock<ILogger<RbacManager>>();
+			var mockTeamsService = new Mock<ITeamsCachedService>();
 			
-			var rbacManager = new RbacManager(BuildConfiguration(), mockUserRoleCachedService.Object, mockLogger.Object);
+			var rbacManager = new RbacManager(mockLogger.Object, mockTeamsService.Object);
```

Comment:
Not a blocker, but I'm not sure this test does what its title suggests - test that users are returned.